svc-license/lcommon/common.go

```diff
@@ -0,0 +1,209 @@
+//(C) Copyright [2020] Hewlett Packard Enterprise Development LP
+//
+//Licensed under the Apache License, Version 2.0 (the "License"); you may
+//not use this file except in compliance with the License. You may obtain
+//a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//License for the specific language governing permissions and limitations
+// under the License.
+
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/ODIM-Project/ODIM/lib-utilities/common"
+	"github.com/ODIM-Project/ODIM/lib-utilities/config"
+	"github.com/ODIM-Project/ODIM/lib-utilities/errors"
+	"github.com/ODIM-Project/ODIM/lib-utilities/response"
+	"github.com/ODIM-Project/ODIM/svc-license/model"
+	log "github.com/sirupsen/logrus"
+)
+
+//GetAllKeysFromTable fetches all keys in a given table
+func GetAllKeysFromTable(table string, dbtype common.DbType) ([]string, error) {
+	conn, err := common.GetDBConnection(dbtype)
```

Comment:
We are planning to remove GetDBConnection() from lib-utilities. I think we can use GetDBConnection function in lib-persistence-manager directly. @shivacharanms please suggest.